CONTRIBUTING.md

```diff
@@ -39,8 +39,40 @@ Or to be safe, you call apply all the required resources at once:
 
 1. `k-apply-all`
 
+### Releasing a New Version
+
+1. Create a new release branch prefixed with `release-` followed by the version number and push the branch to GitHub.
+
+   ```shell
+    git checkout -b release-0.0.1
+    git push origin release-0.0.1
+    ```
+
+1.
+Visit [Tag Workflow Action](https://github.com/crossplane-contrib/provider-upjet-digitalocean/actions/workflows/tag.yaml)
+and click on the `Run workflow` button. Make sure the "Branch" is set to the release branch, e.g., `release-0.0.1`.
+Enter the version number in the "Release version" field, e.g., `0.0.1`, and a sensitive value for the "Tag message"
+field. Verify the information and click the "Run workflow" button.
+1. Wait for the Tag Workflow Action to complete.
+1. Create a new GitHub Release
```

Comment:
typically, we've had to kick off the `CI` action workflow here too, so the CI will run on the newly tagged commit.

Reply:
Wouldn't the following run when we push something to the release branch? https://github.com/crossplane-contrib/provider-upjet-digitalocean/blob/2af60d386b550739c06ab9a423d1b0738519dab1/.github/workflows/ci.yml#L7

Reply:
yes for commits, but I don't think it does after running the `tag` action 🤔